src/static/js/modules/step-your-information.js

```diff
@@ -222,169 +222,81 @@ function init() {
-          $( '#additional-consumer-identity-option3' ).text( 'Co-signer' );
-          $( '#additional-consumer-identity-option4' ).remove();
-          // NOTE THAT THIS DOES NOT WORK FOR LEASES !!
-          break;
-
-        case 'consumer_loan':
-          $( '#primary-consumer-identity-option1' ).text( 'Primary borrower' );
-          $( '#primary-consumer-identity-option2' ).text( 'Co-borrower' );
-          $( '#primary-consumer-identity-option3' ).text( 'Co-signer' );
-          $( '#primary-consumer-identity-option4' ).remove();
-          $( '#additional-consumer-identity-option1' ).text( 'Additional account holder' );
-          $( '#additional-consumer-identity-option2' ).text( 'Co-borrower' );
-          $( '#additional-consumer-identity-option3' ).text( 'Co-signer' );
-          $( '#additional-consumer-identity-option4' ).remove();
-          break;
-
-        case 'card':
-          $( '#primary-consumer-identity-option2' ).text( 'Joint account holder' );
-          $( '#primary-consumer-identity-option3' ).text( 'Additional card holder' );
-          $( '#primary-consumer-identity-option4' ).remove();
-          $( '#additional-consumer-identity-option2' ).text( 'Joint account holder' );
-          $( '#additional-consumer-identity-option3' ).text( 'Additional card holder' );
-          $( '#additional-consumer-identity-option4' ).remove();
-          break;
-
-        case 'checking':
-          $( '#primary-consumer-identity-option1' ).text( 'Joint account holder' );
-          $( '#primary-consumer-identity-option2' ).text( 'Other' );
-          $( '#primary-consumer-identity-option3' ).remove();
-          $( '#primary-consumer-identity-option4' ).remove();
-          $( '#additional-consumer-identity-option1' ).text( 'Joint account holder' );
-          $( '#additional-consumer-identity-option2' ).text( 'Other' );
-          $( '#additional-consumer-identity-option3' ).remove();
-          $( '#additional-consumer-identity-option4' ).remove();
-          break;
-
-        case 'credit_reporting':
-          $( '#consumer1-identity' ).hide();
-          $( '#consumer2-identity' ).remove();
-          break;
-
-        case 'debt':
-          $( '#consumer1-identity' ).hide();
-          $( '#consumer2-identity' ).remove();
-          break;
-
-        default:
-          $( '#primary-consumer-identity-option1' ).text( 'Primary account holder' );
-          $( '#primary-consumer-identity-option2' ).text( 'Additional account holder' );
-          $( '#primary-consumer-identity-option3' ).text( 'Joint account holder' );
-          $( '#primary-consumer-identity-option4' ).text( 'Additional card holder' );
-          $( '#primary-consumer-identity-option5' ).text( 'Co-signer' );
-          $( '#primary-consumer-identity-option6' ).text( 'Co-borrower' );
-          $( '#primary-consumer-identity-option7' ).text( 'Sender' );
-          $( '#primary-consumer-identity-option8' ).text( 'Recipient' );
+          console.log( product )
```

Comment:
Is the `console.log` still needed? (Not that the prototype is running in browsers that don't support it, so leaving it shouldn't cause any trouble.)